tests/phpunit/api/v3/ProfileTest.php

```diff
@@ -109,6 +112,29 @@ public function testProfileGet() {
     }
   }
 
+  /**
+   * Test retrieving a profile with an address custom field in it.
+   *
+   * We are checking that there is no error.
+   *
+   * @throws \CRM_Core_Exception
+   */
+  public function testProfileGetWithAddressCustomData() {
+    $this->_createIndividualContact();
+    $this->entity = 'Address';
+    $this->createCustomGroupWithFieldOfType(['extends' => 'Address']);
+    $this->callAPISuccess('UFField', 'create', [
+      'uf_group_id' => $this->_profileID,
+      'field_name' => $this->getCustomFieldName('text'),
+      'visibility' => 'Public Pages and Listings',
+      'label' => 'My custom field',
+      'field_type' => 'Contact',
+    ]);
+    $this->callAPISuccess('Address', 'get', ['contact_id' => $this->_contactID, 'api.Address.create' => [$this->getCustomFieldName('text') => 'my field']]);
+    $result = $this->callAPISuccess('Profile', 'get', ['profile_id' => $this->_profileID, 'contact_id' => $this->_contactID])['values'];
+    // $this->assertEquals('my field', $result[$this->getCustomFieldName('text')]);
```

Comment:
@eileenmcnaughton are you able to re-enable this last line?

Reply:
@seamuslee001 no - cos it still doesn't pass

Reply:
but the line before failed without @VangelisP's patch merged & I can't figure out the rest

Reply:
Even tho you have merged the other PR?

Reply:
yes - the line before that is the one the other PR fixed